RevenueCatUI/CustomerCenter/Views/PromotionalOfferView.swift

```diff
@@ -62,6 +62,12 @@ struct PromotionalOfferView: View {
 
             VStack {
                 if self.viewModel.error == nil {
+
+                    AppIconView()
+                        .padding(.top, 100)
```

Comment:
Since this padding is always applied, the padding will still be present even if the AppIconView() is just an EmptyView(), which could make the layout look off if the app icon can't be found for some reason. I'd consider making a static property on AppIconView called `canDisplay` and then conditionally setting the top padding based on that value, like:

```swift
.padding(.top, AppIconView.canDisplay ? 100 : 0)
```

Reply:
Oh, I think you switched the missing case from being an EmptyView() to a Color.clear while I was reviewing. Ignore this comment since we want to preserve the space!

Reply:
yeah sorry for the switcheroo, I had uncommitted code. I guess I'm starting to forget how to do PRs properly